app/src/main/java/com/dsciitp/shabd/Category/CategoryFragment.java

```diff
@@ -57,8 +53,7 @@ public static CategoryFragment newInstance(String param1, String param2) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getArguments() != null) {
-            mParam1 = getArguments().getString(ARG_PARAM1);
-            mParam2 = getArguments().getString(ARG_PARAM2);
+            position = getArguments().getInt(ARG_PARAM1);
```

Comment:
Use proper variable name. Use something that is meaningful and indicates what the variable is doing.

Reply:
Well, its not doing anything for now, its default and will be used in future

Reply:
Also, TODO is added for changing parameter names